frontend/src/components/PlayerPage.tsx

```diff
@@ -45,37 +65,38 @@ export default function PlayerPage() {
                     <StyledText>Lat: {player.coordinates.latitude}</StyledText>
                     <StyledText>Lon: {player.coordinates.longitude}</StyledText>
                 </StyledPlayerCoordinates>
-                <StyledPlayerNodes>
-                    <StyledText>Nodes:</StyledText>
-                    <StyledText>{ }</StyledText>
-                </StyledPlayerNodes>
             </StyledCard>
+                <Typography color={theme.palette.success.main} variant={"h5"}>Nodes</Typography>
                 <StyledButtonContainer>
-                    <StyledFormButton theme={"primary"} onClick={() => navigate("/")}>Nodelist</StyledFormButton>
-                    <StyledFormButton theme={"error"} onClick={() => logout(navigate)}>Logout</StyledFormButton>
+                    <StyledFormButton theme={"primary"} onClick={() => {
+                        playClick()
+                        navigate("/")
+                    }}>Nodelist</StyledFormButton>
+                    <StyledFormButton theme={"error"} onClick={() => logout(navigate, playSuccess, playError)}>Logout</StyledFormButton>
                 </StyledButtonContainer>
+                {nodes && <NodeList player={player} nodes={nodes}/>}
         </>
         )
     }
 }
 
 const StyledCard = styled(Card)`
-  margin-top: 1rem;
+  margin: 1rem 0;
   width: 95%;
-  height: 75vh;
+  height: 50vh;
   background: var(--color-semiblack);
   display: grid;
   grid-template-columns: 1fr 1fr;
-  grid-template-rows: 2fr 1fr 2fr 2fr;
+  grid-template-rows: 2fr 1fr 2fr;
 `;
 
-const StyledAvatar = styled(Avatar)<{isEnemy: string}>`
+const StyledAvatar = styled(Avatar)<{isenemy: string}>`
   grid-row: 1;
   grid-column: 1;
   display: flex;
   margin: auto;
-  border: 2px solid ${props => props.isEnemy === "true" ? "var(--color-secondary)" : "var(--color-primary)"};
-  filter: drop-shadow(0 0 0.15rem ${props => props.isEnemy === "true" ? "var(--color-secondary)" : "var(--color-primary)"});
+  border: 2px solid ${props => props.isenemy === "true" ? "var(--color-secondary)" : "var(--color-primary)"};
+  filter: drop-shadow(0 0 0.15rem ${props => props.isenemy === "true" ? "var(--color-secondary)" : "var(--color-primary)"});
```

Comment:
Why did you switched to lowercase?

Reply:
react kept complaining that, if I want to use custom attributes I should make them all lowercase. Weird...

Reply:
I had the same issue actually. Did it solve it for you? Because i think i solved it with parsing all styled Props variable with a $prefix. For example `$isEnemy`

Reply:
Wow I'm so thankful for this hint! Didn't know about the `$` prefix